src/nvim/vim.h

```diff
@@ -27,9 +27,6 @@ Error: configure did not run properly.Check auto/config.log.
 # endif
 #endif
 
-/* user ID of root is usually zero, but not for everybody */
-#define ROOT_UID 0
```

Comment:
Even if the comment is misleading, we should not be using a magic number in the source code...